probe-rs/src/probe/jlink/interface.rs

```diff
@@ -9,24 +9,22 @@ enum_and_set!(
     #[non_exhaustive]
     #[derive(Debug, Copy, Clone, PartialEq, Eq)]
     pub enum Interface {
-        /// JTAG interface (IEEE 1149.1). Supported by most J-Link probes (some embedded J-Links might
-        /// only support SWD).
+        /// JTAG interface (IEEE 1149.1). Supported by most J-Link probes (some embedded J-Links
+        /// might only support SWD).
         Jtag = 0,
-        /// SWD interface (Serial Wire Debug), used by most Cortex-M chips, and supported by almost all
-        /// J-Link probes.
+        /// SWD interface (Serial Wire Debug), used by most Cortex-M chips, and supported by almost
+        /// all J-Link probes.
         Swd = 1,
         /// Background Debug Mode 3, a single-wire debug interface used on some NXP microcontrollers.
         Bdm3 = 2,
         /// FINE, a two-wire debugging interface used by Renesas RX MCUs.
         ///
-        /// **Note**: due to a bug, attempting to select FINE with [`JLink::select_inferface`] will
+        /// **Note**: due to a bug, attempting to select FINE with `JLink::select_interface()` will
```

Comment:
Why remove the link? :)

Reply:
private struct, it doesn't show up in docs

Reply:
link was also broken